tests/data/imds/unicode.json

```diff
@@ -0,0 +1,46 @@
+{
+  "compute": {
+    "location": "wéstus",
+    "name": "héalth",
+    "offer": "UbuntuSérvér",
+    "osType": "Linux",
+    "placementGroupId": "",
+    "platformFaultDomain": "0",
+    "platformUpdateDomain": "0",
+    "publisher": "Canonical",
+    "resourceGroupName": "tésts",
+    "sku": "16.04-LTS",
+    "subscriptionId": "21b2dc34-bcé6-4é63-9449-d2a8d1c2339é",
+    "tags": "",
+    "version": "16.04.201805220",
+    "vmId": "é7fdbfc4-2déb-4a4é-8615-éa6aaf50162é",
```

Comment:
Why are they fancy in the GUID?

Reply:
No special reason - just did a find/replace for the unit test